etc/HIReplayOfflineConfiguration.py

```diff
@@ -136,8 +149,8 @@
 alcarawProcVersion = dt
 
 # Defaults for GlobalTag
-expressGlobalTag = "132X_dataRun3_Express_v4"
-promptrecoGlobalTag = "132X_dataRun3_Prompt_v4"
+expressGlobalTag = "141X_dataRun3_Express_v2"
+promptrecoGlobalTag = "141X_dataRun3_Prompt_v2"
```

Comment:
```suggestion
promptrecoGlobalTag = "141X_dataRun3_Prompt_v3"
```
`141X_dataRun3_Prompt_v3` has `DeDxCalibrationRcd`, see https://github.com/cms-sw/cmssw/issues/46186